### ISSUE TITLE
Build error...

### ISSUE THREAD

**Issue by smblott-github** [ligfietser/mkgmap-style-sheets#3]
Hi,

I see this build error when trying to use these styles...

```
SEVERE (StyledConverter): data/switzerland/pbf/75310013.osm.pbf: routable type 0x08 is used with a non-routable way which was also added as a routable way. This leads to routing errors. Try --check-styles to check the style
```

I guess that's not normal.  Is this user error, or is there a problem with the styles?

Mkgmap version 3676.

**Comment by ligfietser** [comment]
Hi,
Yes I am aware of this warning, you can ignore this error, this linetype 0x08 is also used for dams that are not a highway, it will not lead to routing errors because there are no addresses involved that are connected to those dams. However if you don't like those warnings, you can replace those lines with a non routable line type.

**Comment by smblott-github** [comment]
Ta!
